docs/components.md

```diff
@@ -36,15 +36,15 @@ support at this time). While in alpha, breaking changes may be introduced in a
 new release. In addition, any of these components may be removed prior to the
 1.0 release.
 
-| Receivers                                                                                                                 | Processors | Exporters                                                                                           | Extensions |
```

Comment:
do you know why it's marking all these lines as deleted and then added?

Reply:
I think it's because I changed some of the spacing to try to line the columns up in the plain-text. If we don't want whitespace changes in this commit I can remove my changes.

Reply:
I've removed unnecessary whitespace changes.